tests/test_questions/test_view_question.py

```diff
@@ -47,3 +54,41 @@ def test_paginated_question_invalid_page_query(self):
             get_paginated_question_invalid_page,
             "Page does not exist"
         )
+
+    def test_all_questions_query(self):
+        """
+        Test getting all questions
+        """
+        CommonTestCases.admin_token_assert_equal(
+            self,
+            all_questions_query,
+            all_questions_query_response
+        )
+
```

Comment:
Also, write the test for when there is response for Question not found

Reply:
That test is already included. I used `assert_in` for that test and therefore do not need to create a response fixture for that.

Reply:
ok